src/ui/Base/Initialization.ts

```diff
@@ -725,9 +725,9 @@ export class LazyInitialization {
 
   public static buildErrorCallback(chunkName: string, resolve: Function) {
     return error => {
-      LazyInitialization.logger.warn(
-        `Cannot load chunk for ${chunkName}. You may need to configure the paths of the resources using Coveo.configureResourceRoot. Current path is ${__webpack_public_path__}.`
-      );
+      LazyInitialization.logger
+        .warn(`Cannot load chunk for ${chunkName}. You may need to add the coveo-script class on the script tag that includes the Coveo framework. More details here https://docs.coveo.com/en/418/javascript-search-framework/javascript-search-framework-version-1-x-to-2-x-breaking-changes-and-migration-guidelines#fixing-code-chunks-loading-path-issues
```

Comment:
I'll ask Francois if it's the correct manner to create link on the new doc site, or if there are ways to create shortened URL.

Also, it seems there's nothing that talks about troubleshooting issues for chunk loading in that page.

Reply:
Ok, seems like you can simply use the part of the URL up to the ID.

So : https://docs.coveo.com/en/418 .

The ID is permanent, and will not break if the article is moved.

Reply:
Yea I used the wrong link, I wanted it to be https://docs.coveo.com/en/295/javascript-search-framework/lazy-versus-eager-component-loading#fixing-code-chunks-loading-path-issues so it would scroll to the relevant section.

But if using the full link risks breaking I guess I'll use the one with id only

Reply:
using the full URL won't break. You can keep it. If it's not in the page, it will just open the page normally and will still work, since it's added client side apparently.